### PR TITLE
Optimize root selector transformation

### PR DESCRIPTION
**Pull request by dvoytenko**

### CODE REVIEW

File: src/shadow-embed.js [hunk 1]
```diff
@@ -260,19 +260,32 @@ export function scopeShadowCss(shadowRoot, css) {
...
+ * @param {string} match
+ * @param {string} name
+ * @param {number} pos
+ * @param {string} selector
+ * @return {string}
+ * @private
+ */
+function rootSelectorPrefixer(match, name, pos, selector) {
+  const prev = pos > 0 ? selector.charAt(pos - 1) : '';
+  const next = pos + match.length < selector.length ?
```
Comment (jridgewell) [inline src/shadow-embed.js]: This ternary isn't necessary, `#charAt` will return `''` if it's out of bounds.

File: src/shadow-embed.js [hunk 2]
```diff
@@ -260,19 +260,32 @@ export function scopeShadowCss(shadowRoot, css) {
...
+ * See `transformRootSelectors`.
+ * @param {string} match
+ * @param {string} name
+ * @param {number} pos
+ * @param {string} selector
+ * @return {string}
+ * @private
+ */
+function rootSelectorPrefixer(match, name, pos, selector) {
+  const prev = pos > 0 ? selector.charAt(pos - 1) : '';
```
Comment (jridgewell) [inline src/shadow-embed.js]: Same.

**Comment by jridgewell** [comment]
Nitpicks, LGTM.

**Comment by dvoytenko** [comment]
@jridgewell both done.

**Comment by dvoytenko** [comment]
@jridgewell both done.

**Pull request merged**